pom.xml

```diff
@@ -181,7 +181,7 @@
     <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
     <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
     <fasterxml.jackson.databind.version>2.15.0</fasterxml.jackson.databind.version>
-    <snappy.version>1.1.9.1</snappy.version>
+    <snappy.version>1.1.10.0</snappy.version>
```

Comment:
Also update https://github.com/apache/spark/blob/master/dev/deps/spark-deps-hadoop-3-hive-2.3#L237?

Reply:
oops forgot about this!